toolkit/scripts/toolchain/download_toolchain_rpm.sh

```diff
@@ -19,6 +19,8 @@ function usage() {
     echo "           until the RPM is successfully downloaded. Full URL will be <url>/<rpm_name>"
     echo "-c|--certificate: Optional path to a certificate file to use for the download"
     echo "-k|--private-key: Optional path to a private key file to use for the download"
+    echo "-K|--enforce-signatures: Optional flag to enforce RPM signatures"
```

Comment:
I would choose a different letter to avoid confusion since 'k' is already used

Reply:
I was basing on `rpm -K`, but I'll think of a better one.